packages/react-conformance/src/utils/getComponent.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactWrapper } from 'enzyme';
+import { Ref, RefFindNode } from '@fluentui/react-component-ref';
```

Comment:
need to add `@fluentui/react-component-ref` as dependency in package.json

Reply:
Good catch, I read straight over this. It's probably why the build was failing (though I'm not sure why it failed with the error messages it did...). 

I think that rather than adding a dep on react-component-ref and directly referencing these components, it would be more cross-package-friendly to make `wrapperComponent` an array (`wrapperComponents`) and pass in the components that way. That will be the solution for this PR.

In a separate PR, I think at this point it also makes sense to add a new utility under `packages/fluentui/react-northstar/test/utils` which wraps the new `isConformant` and mixes in options which will always be applicable (such as `wrapperComponents: [Ref, RefFindNode]`, or certain test disables if needed). And a similar utility for OUFR and react-next, probably under `src/common` in each package.